canonical-web-and-design/ubuntu.com
# Release free trials

## Done

- Implements free trials on the UA shop

## QA

- go to https://ubuntu-com-10378.demos.haus/advantage/subscribe?test_backend=true
- see that you can start a guest trial
- log in
- start a trial
- check that it appears on /advantage
- try to add another subscription and see that you can't while your trial is active
- try to end your trial and buy (card number: 42424242424242424....)
- try to start another trial with the same product (you shouldn't be eligible)
- try to break stuff
